README.md

```diff
@@ -69,6 +69,32 @@ multi-network-policy-tc runs as a daemonset on each node.
 `multi-networkpolicy-tc` watches MultiNetworkPolicy object and creates TC rules on VF representor to filters packets
  to/from interface, based on MultiNetworkPolicy.
 
+## Configuration reference
+
+The following configuration flags are supported by `multi-networkpolicy-tc`:
+```shell
```

Comment:
nit: maybe remove "shell"? it leads a weird coloring

Reply:
done